plugins/woocommerce/client/legacy/css/_variables.scss

```diff
@@ -11,7 +11,9 @@ $blue:          	#2ea2cc !default;
 $primary:           #720eec !default;                                    // Primary color for buttons (alt)
 $primarytext:       desaturate(lighten($primary, 50%), 18%) !default;    // Text on primary color bg
 
-$secondary:         desaturate(lighten($primary, 40%), 21%) !default;    // Secondary buttons
+// We use the old WooCommerce color as the base of the secondary color on purpose.
+// See: https://github.com/woocommerce/woocommerce/issues/55165.
+$secondary:         desaturate(lighten(#7F54B3, 40%), 21%) !default;    // Secondary buttons
```

Comment:
Will reverting to the original color affect the branding side of things? Asking because there must be a reason were updated it in the first place.

Reply:
Looking at it now, I think the secondary color should have been excluded from the rebranding. As https://github.com/woocommerce/woocommerce/issues/55165 illustrates, people assumed it was a gray completely disconnected from the WooCommerce purple. The secondary color was mostly used in the frontend, so any changes to it ended up impacting the frontend of existing stores.

In the PR, I left the calculation `desaturate(lighten(#7F54B3, 40%), 21%)`, but I think we could perfectly replace it with the computed value (`#e9e6ed`) to make it clearer that this color is disconnected from the WooCommerce purple.

Reply:
Thanks for the context, it will be helpful to leave a comment so that we don't touch it in the future :)

Reply:
Ok, I will do that in `trunk` if that's fine, so we don't delay the release of this fix. :slightly_smiling_face: